.env.test

```diff
@@ -8,4 +8,11 @@ user_name=Mongo_test_user
 password=zk1FDHusxzukaTxo
 mongo_url=shdevopscluster.4jxwd.mongodb.net
 database_name=ToDoApp_DB_test
-collection_name=todoapp_collection_test
\ No newline at end of file
+collection_name=todoapp_collection_test
+
+#OAuth Connection details
+client_id=6f99afcsjkfif39e6783
+client_secrets=4sksie938frjf949fk409gfa0f1d41cd4c7634f97

```

Comment:
These look like they could be genuine, if they are we should rotate the secret and remove them from here! Our test environment should disable login so that these don't need to be real (and if they aren't, feel free to ignore me, but I'd always recommend making that really clear like `client_id=fake-id` to save other developers from worrying!)

Reply:
No it is not genuine, it is fake client id and client secret. As advised I have amended them to 
client_id=fake-client-id
client_secrets=fake-client-secret